cdap-support-bundle/src/main/java/io/cdap/cdap/support/handlers/SupportBundleHttpHandler.java

```diff
@@ -48,6 +67,11 @@
 @Path(Constants.Gateway.API_VERSION_3)
 public class SupportBundleHttpHandler extends AbstractHttpHandler {
 
+  private static final Logger LOG = LoggerFactory.getLogger(SupportBundleHttpHandler.class);
```

Comment:
Unused?

Reply:
Used for the base directory is not existed